packages/react/src/forms/Radio/Radio.module.css

```diff
@@ -8,6 +8,7 @@
   -ms-user-select: none;
   user-select: none;
   cursor: pointer;
+  grid-row: 1;
```

Comment:
It's necessary to add this here because we updated the styles in the FormControl--checkbox class to use flex-grid and this class gets applied to both Checkbox and Radio inputs that are children of a FormControl component.